docs/en/04_Performance_Guide/02_HTTP_Caching.md

```diff
@@ -120,39 +131,58 @@ Full caching can only be achieved on dynamic content if that content is non-vary
 Be cautious! If you feel uncertain about identifying content as non-varying, better stick to "Light" caching and avoid the danger of leaking user-specific or confidential data altogether.
 </div>
 
-```
-Injector:
-  FullCachingPolicy:
-    class: CachingPolicy
-    properties:
-      cacheAge: <cache-duration>
-      vary: ''
-MyFullyCached_Controller:
-  dependencies:
-    Policies: '%$FullCachingPolicy'
+```php
+<?php
+
+use CWP\CWP\PageTypes\BasePageController;
+
+class FullyCachedPageController extends BasePageController
+{
+    protected function init()
+    {
+        parent::init();
+        
+        HTTPCacheControlMiddleware::singleton()
+            ->enableCache()
+            ->setVary([])
```

Comment:
I wonder if we should recommend at least 'Vary: Accept-Encoding' even in full caching?

Reply:
This should be set/added by any layer that changes the encoding. mod_gzip and mod_deflate add the vary header automatically (for example)

Reply:
yeah, that is true, I haven't tested it just yet in our infrastructure to ensure that it is correct, but I have no reason to think that it isn't working.